app/src/main/java/edu/grinnell/kdic/RadioService.java

```diff
@@ -143,20 +138,17 @@ public void onAudioFocusChange(int focusChange) {
 
                     case AudioManager.AUDIOFOCUS_LOSS:
                         // Lost focus for an unbounded amount of time: stop playback and release media player
-                        Log.d(TAG, "AudioManager: AUDIOFOCUS_LOSS");
                         if (isPlaying()) reset();
                         break;
 
                     case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT:
-                        Log.d(TAG, "AudioManager: AUDIOFOCUS_LOSS_TRANSIENT");
                         // Lost focus for a short time, but we have to stop
                         // playback. We don't release the media player because playback
                         // is likely to resume
                         if (isPlaying()) pause();
                         break;
 
                     case AudioManager.AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK:
-                        Log.d(TAG, "AudioManager: AUDIOFOCUS_LOSS_TRANSIENT_CAN_DUCK");
                         // Lost focus for a short time, but it's ok to keep playing
                         // at an attenuated level
                         if (isPlaying()) mediaPlayer.setVolume(0.2f, 0.2f);
```

Comment:
Extract the float values as constants and name them `MEDIA_PLAYER_LEFT_VOLUME` and `MEDIA_PLAYER_RIGHT_VOLUME` accordingly.

See: https://developer.android.com/reference/android/media/MediaPlayer.html#setVolume(float, float) for choice of constant naming